webapp/components/topbar/dialogs/LaunchDialog.js

```diff
@@ -267,7 +267,7 @@ const LaunchDialog = (props) => {
   return (
     <ActionDialog
       buttonLabel={LAUNCH_MODAL.actionSimulate}
-      title={LAUNCH_MODAL.actionSimulate}
+      title={LAUNCH_MODAL.actionSimulate + " " + props.experimentInDesign.name}
```

Comment:
eslint should show you here some errors, use template string instead of concatenation

Reply:
In line 240 is a props destructuring statement `const { classes } = props;`
Add `experimentInDesign` to it

```
const { classes, experimentInDesign } = props;
```